Budgets section

DzhideX opened the pull request:
Implement new Budgets section as discussed in this issue: https://github.com/Joystream/joystream/issues/4534

Thanks @toiletgranny for providing new designs for the budget amounts :heart:

bedeho reviewed (changes requested):
This looks good, well done!

Why is there still a `useAirtableData.js` file here though, we should not be getting any data from any Airtable at this point.

DzhideX commented:
> This looks good, well done!
 
 Thank you! :raised_hands: 
 
 > Why is there still a useAirtableData.js file here though, we should not be getting any data from any Airtable at this point.
 
 Indeed we aren't! I was just using that file to store static data to help with rendering the UI and forgot to move it to a more fitting location. That fits better inside of `src/data/pages/index.js` so I moved it there now and also deleted the `useAirtableData.js` file.
